all/099-ceph.yml

```diff
@@ -2,6 +2,9 @@
 ##########################
 # generic
 
+monitor_interface: "{{ network_interface }}"
+radosgw_interface: "{{ network_interface }}"
```

Comment:
That's the wrong file. Use 099-interfaces.yml.

Reply:
Ah, i haven't expected that the interfaces are centralized. Fixed.